### DISCUSSION TITLE
Error loading the 'sqlite3' Active Record adapter

### DISCUSSION THREAD

**Discussion by ZuluPilgrim** [pglombardo/PasswordPusher#934] [Q&A] [answered]
I installed using:  bundle config set with 'mysql'

I installed MySQL beforehand but I am still getting the following.

`sysadmin@pwdpusher:~/PasswordPusher$ bundle exec rake assets:precompile
rake aborted!
LoadError: Error loading the 'sqlite3' Active Record adapter. Missing a gem it depends on? sqlite3 is not part of the bundle. Add it to your Gemfile.
/home/sysadmin/PasswordPusher/config/environment.rb:5:in `<main>'
/home/sysadmin/.rbenv/versions/3.2.0/bin/bundle:25:in `load'
/home/sysadmin/.rbenv/versions/3.2.0/bin/bundle:25:in `<main>'

Caused by:
Gem::LoadError: sqlite3 is not part of the bundle. Add it to your Gemfile.
/home/sysadmin/PasswordPusher/config/environment.rb:5:in `<main>'
/home/sysadmin/.rbenv/versions/3.2.0/bin/bundle:25:in `load'
/home/sysadmin/.rbenv/versions/3.2.0/bin/bundle:25:in `<main>'
Tasks: TOP => environment
(See full trace by running task with --trace)`

**Comment by pglombardo** [comment, marked as answer]
Hi @ZuluPilgrim - I'll delete the discussion since you've moved on from this issue.  But for reference, make sure to always set `RAILS_ENV` to avoid the default fall back to sqlite3.
